docs/tutorial.md

```diff
@@ -1,13 +1,19 @@
 # Tutorial
 
+## Important note:
```

Comment:
I would remove "In this new version we've added". I would also remove this section and move these links to [The point where we create the integration and we choose a local entity](https://github.com/newrelic/infra-integrations-sdk/pull/116/files#diff-0adced32de34cd29581087ebf0482058R137), briefly explaining why it has sense to choose a local entity.

Reply:
I've rephrased it but I think this section is important to avoid people looking for multiple entities go through all the tutorial which is not short.